CONTRIBUTING.md

```diff
@@ -1,151 +1,132 @@
 Marionette has a few guidelines to facilitate your contribution and streamline
-the process of getting the changes merged in and released.
+the process of getting changes merged in and released.
 
-## Running the tests
+1. [Setting up Marionette locally](#setting-up-marionette-locally)
+2. [Reporting a bug](#reporting-a-bug)
+  1. [When you have a patch](#when-you-have-a-patch)
+  2. [When you don't have a bug fix](#when-you-dont-have-a-bug-fix)
+3. [Running Tests](#running-tests)
+4. [Updating docs](#updating-docs)
 
-There are two methods to running the unit test suite, suited for your desired workflow and overall developer happiness:
-you can open SpecRunner.html in the browser or run `npm test`
-To begin ensure that you have installed the project dev dependencies via:
 
-    $ npm install
+## Setting up Marionette locally
 
-### In a browser
+* Fork the Marionette repo.
 
-  Open [SpecRunner.html](SpecRunner.html) in your browser.
+* `git clone` your fork onto your computer.
 
-### Via Grunt
+* Run `npm install` to make sure you have all Marionette dependencies locally.
 
-  Running `grunt` alone will default to running the tests once.
-  Running `grunt watch` will rerun the tests after any file change.
+## Reporting a bug
 
-## Documentation Fixes
+In order to best help out with bugs, we need to know the following information
+in your bug submission:
 
-If you notice any problems with any documentation, please
-fix it and we'll get it merged as soon as we can. For
-small things like typos and grammar (which we know I'm
-terrible with), just click the "Edit this file" button
-and send in the pull request for the fix. For larger
-changes and big swaths of documentation changes, a regular
-pull request as outlined below is more appropriate.
+* Marionette version no.
+* Backbone version no.
+* jQuery version no.
+* Underscore version no.
+* Browser and version no.
+* Operating System and version no.
```

Comment:
The version requirements seem pretty dated. I'd say, we only care about Mn and BB. 90% of the time. I'd like to see the others dropped because they add a level of intimidation (oh no, maybe my bug happened bc I'm running leopard)